### DISCUSSION TITLE
İBXC PUANLARININ KARŞILIĞI

### DISCUSSION THREAD

**Discussion by Emirxt** [IBAX-io/go-ibax#78] [Rewards]
BU GÖREVLERDEN KAZANDIĞIMIZ PUANLARIN KARŞILIĞI NE KADAR OLACAK?

**Comment by Majezikonline** [comment]
Bende merak ediyorum gerçekten👍

**Reply by Tunc37** [reply]
### puan karşılığından ziyade projenin geleceği konusunda detaylı olarak nereden bilgi alacağız

**Comment by Hasan434343** [comment]
Balon olmasın

**Comment by hemresenol** [comment]
Altyapıları gayet sağlam avaxı uzun vadede geçebilir puanlarda tatmin edici bir ibaxa dönüşebilir.

**Comment by byblckd** [comment]
https://ibax.io/user/airdrop-rules
bu sayfada puan dağıtım şekli detaylı şekilde anlatılmış.Kontrol ederseniz sizleri tatmin edecek sonuçlara erişebiliriz

**Comment by cihanaslan0** [comment]
Sizce ilk listelendiğinde fiyatı ne olur?

**Reply by byblckd** [reply]
2.1 milyar adet coinden bahsediyoruz alacağı yatırım projenin dikkat çekmesi kullanacağı teknoloji ve bir dünya şey fiyatı etkileyebilir.Benim tahminim 210 milyon dolar piyasa değeriyle piyasaya girse bile edeceği coin başı 0.1 dolar olur

**Comment by cartvecurt** [comment]
Toplamda dağıtılacak bir ödül miktarı var, en son toplanan puanlara orantılı şekilde bir dağıtım olur, şimdiden şu kadar demek zor.

**Comment by alisabanoglu** [comment]
İlerisi için avax a rakip olabilir
